src/test/java/de/zalando/sprocwrapper/proxy/SubVirtualShardKeyStrategy.java

```diff
@@ -0,0 +1,10 @@
+package de.zalando.sprocwrapper.proxy;
+
+import de.zalando.sprocwrapper.sharding.VirtualShardKeyStrategy;
+
+/**
+ * @author Soroosh Sarabadani
+ */
+public class SubVirtualShardKeyStrategy extends VirtualShardKeyStrategy {
```

Comment:
Sorry, I was wrong. It is already in the test scope. Somehow did not notice that :D